src/python/pants/option/global_options.py

```diff
@@ -1397,7 +1397,7 @@ class BootstrapOptions:
         default=None,
         advanced=True,
         deprecation_start_version="2.15.0.dev2",
-        removal_version="2.16.0.dev1",
+        removal_version="2.16.0.dev2",
```

Comment:
@asherf this was a bit too hairy for me, I gave it a shot but ran away scared. I'
m not sure if this is something you can take up soon, but the next releaser would likely be appreciative.